.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
```

Comment:
Seems fair that we can do this with labels, just might be easy to forget about it. I don't mind too much either way since it's probably clear from the description what kind of ticket it is. This sort of thing might just be good for deciding which version number to use but I don't think we're using semantic versioning right now anyway.

I would like to see some indication that tests pass (until we get jenkins testing up and running) so I removed the table and added a checkbox at the bottom.